src/app/interceptors/http/bad-request.interceptor.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+
+import { Observable, throwError } from 'rxjs';
+import { catchError, filter } from 'rxjs/operators';
+
+export class BadRequestHttpInterceptor implements HttpInterceptor {
```

Comment:
Sort methods alphabetically

Reply:
import { catchError, filter } from 'rxjs/operators'; **first**?
import { Observable, throwError } from 'rxjs';

how do you handle when you have **rxjs/operators** and **rxjs**?
What is the benefit of being very specific in its importation?

Reply:
import { Observable, throwError } from 'rxjs'
import { catchError, filter } from 'rxjs/operators';